sdk/openai/openai/test/public/samples.spec.ts

```diff
@@ -6,7 +6,7 @@ import { assert } from "@azure/test-utils";
 import { Context } from "mocha";
 import { OpenAIClient } from "../../src/index.js";
 import { createClient, startRecorder } from "./utils/recordedClient.js";
-
+import { ImageLocation } from "../../src/models/index.js";
```

Comment:
nitpick: import from the top-level index directly so we can easily enable min/max testing in the future:
```suggestion
import { ImageLocation } from "../../src/index.js";
```